src/sc2laddercore/LadderConfig.h

```diff
@@ -1,5 +1,12 @@
 #pragma once
 
+#define RAPIDJSON_HAS_STDSTRING 1
+#include "rapidjson.h"
+#include "document.h"
+
+#include <string>
+#include <map>
+
 class LadderConfig
 {
 public:
```

Comment:
Please use explicit in the constructor.

Reply:
Any specific reason for this? or is this just good practice?

Reply:
There are number of caveats behind, see:
https://stackoverflow.com/questions/12437241/c-always-use-explicit-constructor/12437453

Reply:
Thanks for the info.